dolphinscheduler-dao/src/main/resources/sql/dolphinscheduler_mysql.sql

```diff
@@ -2007,3 +2010,25 @@ CREATE TABLE `t_ds_fav_task`
 ) ENGINE = InnoDB
   AUTO_INCREMENT = 1
   DEFAULT CHARSET = utf8;
+
+-- ----------------------------
+-- Table structure for t_ds_task_remote_host
+-- ----------------------------
+DROP TABLE IF EXISTS `t_ds_task_remote_host`;
+CREATE TABLE `t_ds_task_remote_host`
```

Comment:
I do not think adding a table for a task type is a good idea, can we ship it to our data source center? I know that we will change its name from `datasource center` to `connection center`, which can have various types of connection instead of database only

Reply:
If we add it to `connection center`, we can also migrate your `Add a Task Remote Host Management page in Security` into `connection center`, which may keep our web UI simple and unify

Reply:
> > Add a Task Remote Host Management page in Security
> 
> If we add it to `connection center`, we can also migrate it into `connection center`, which may keep our web UI simple and unify

yes, but the conclusion we discussed bi-week was to temporarily in `Security`, after `Connection center` fine, we will migrate it into `Connection center`